src/js/components/HeaderBar.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from "react";
 import { Link } from "react-router";
 import { MountService } from "foundation-ui";
+import { Tooltip } from "@dcos/ui-kit";
 
 import ClusterHeader from "./ClusterHeader";
 import SidebarToggle from "./SidebarToggle";
+import Config from "../config/config";
```

Comment:
this path does not work on all unixes. your mac seems to handle paths case insensitive. you'll want to use `"../config/Config"`.